dts/bindings/vendor-prefixes.txt

```diff
@@ -490,6 +490,7 @@ okaya	Okaya Electric America, Inc.
 oki	Oki Electric Industry Co., Ltd.
 olimex	OLIMEX Ltd.
 olpc	One Laptop Per Child
+omron	Omron Corporation
```

Comment:
@nashif please help review this prefix

Reply:
Changed to `OMRON` due to [this comment](https://github.com/zephyrproject-rtos/zephyr/pull/96284#discussion_r2363655407)